### DISCUSSION TITLE
Consider releasing 1.1.2

### DISCUSSION THREAD

**Discussion by Maronato** [plausible/analytics#514] [Self-Hosted Support]
I know that it’s really not my business when you publish your releases, but 1.1.2 is getting pretty big for a patch release, according to semantic versioning. Maybe consider publishing it and/or changing it to a minor or even major?

**Comment by metmarkosaric** [comment]
thanks @Maronato! we've published more details on the self-hosting release schedule here: https://plausible.io/blog/building-open-source (tl;dr: basically we plan to release it twice a year)

**Reply by ukutaht** [reply]
@Maronato Yeah, we'll make it a minor release since there are a ton of new features. Targeting Jan 2021.
